presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -148,4 +160,11 @@ public ColumnStatistics build()
             return new ColumnStatistics(nullsFraction, distinctValuesCount, dataSize, lowValue, highValue);
         }
     }
+
+    private static void checkArgument(boolean test, String message, Object... args)
```

Comment:
Maybe you could extract to some `Preconditions` class in spi?

Reply:
We got a lot of `checkArgument` and `vefiry` methods implementation across the `presto-spi` module. I think that a good idea to have a `Precondition` class in that module to avoid implementing this boilerplate code every time someone needs that. However to keep it consistent i would rather like to do this as a separate PR, and i would like to replace the usages everywhere.